pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -361,17 +361,42 @@ func (ctrl *PersistentVolumeController) syncUnboundClaim(ctx context.Context, cl
 				return nil
 			}
 
+			// The claim can't be bound to any existing volume. let's check if it can be
+			// dynamically provisioned. If can't, we will emit an event to inform the user
+			// the reason why the claim is pending. otherwise, we will provision a volume
+			// for the claim in an asynchronous goroutine and return immediately.
```

Comment:
"_The claim can't be bound to any existing volume. let's check if it can be dynamically provisioned._" - that repeats "// No PV could be found" above. If that is not enough, then fix the existing comment and don't repeat it elsewhere.

The rest just basically copies the code below. Is it useful to repeat it?